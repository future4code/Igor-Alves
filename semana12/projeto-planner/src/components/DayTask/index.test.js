import React from 'react';
import renderer from 'react-test-renderer';
import { DayTask } from './index'

it('Snapshot DayTask', ()=> {
    const tree = renderer
    .create(<DayTask></DayTask>)
    .toJSON();
    expect(tree).toMatchSnapshot()
})