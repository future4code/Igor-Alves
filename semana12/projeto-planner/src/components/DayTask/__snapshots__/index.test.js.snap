// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot DayTask 1`] = `
<div
  className="sc-bwzfXH dXshDT"
>
  <h3
    className="sc-bdVaJa hoqfrS"
  />
</div>
`;
